### PR TITLE
Fix statefulset absent of kubectl scale in i18n.T

### PR DESCRIPTION
**Pull request by chymy**
Signed-off-by: chymy <chang.min1@zte.com.cn>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func NewCmdScale(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobr
 	cmd := &cobra.Command{
 		Use:                   "scale [--resource-version=version] [--current-replicas=count] --replicas=COUNT (-f FILENAME | TYPE NAME)",
 		DisableFlagsInUseLine: true,
-		Short:                 i18n.T("Set a new size for a Deployment, ReplicaSet or Replication Controller"),
+		Short:                 i18n.T("Set a new size for a Deployment, ReplicaSet, Replication Controller or StatefulSet."),
```
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go:113]: /lgtm cancel
If you want to fix it, let's fix the entire description of this command such that it talks about the possibility to scale any resource having `/scale` sub-resource. As an example calling out the above.

**Review by soltysh — CHANGES_REQUESTED** [review]

File: staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go (hunk 1 above)
Comment (chymy) [reply staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go:113]: No more sub-resources to scale?
`Short` description is consistent with `Long` description
https://github.com/kubernetes/kubernetes/blob/66cbf0196bd5ed0f06d7c40fccba887aae5405d8/staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go#L39-L41

**Pull request closed without merging**